packages/nc-gui/components/virtual-cell/QrCode.vue

```diff
@@ -3,7 +3,7 @@ import { useQRCode } from '@vueuse/integrations/useQRCode'
 import type QRCode from 'qrcode'
 import { RowHeightInj, computed, inject, ref } from '#imports'
 
-const maxNumberOfAllowedCharsForQrValue = 2000
+const maxNumberOfAllowedCharsForQrValue = 62
```

Comment:
how to get this number?

Reply:
i tried entering few characters , when the character count reach 63 qr code is not generating